.pre-commit-config.yaml

```
@@ -0,0 +1,33 @@
+    hooks:
+      - id: actionlint
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.3
+    hooks:
+      # Run the linter.
+      - id: ruff
+        args: [--fix]
+      # Run the formatter.
+      - id: ruff-format
+ci:
+    autofix_commit_msg: |
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 4

(indentation)